### PR TITLE
Bug 1296333, added command to check if Docker is running

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1296333

@adellape I know you just did some work in this section. PTAL and let me know if any changes are needed.

### CODE REVIEW

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -783,6 +789,9 @@ This will destroy any Docker containers or images currently on the host.
 # rm -rf /var/lib/docker/*
 # systemctl restart docker
 ----
++
+If there is any content in *_/var/lib/docker/*_*, it has to be deleted. Files
```
Comment (tpoitras) [inline install_config/install/prerequisites.adoc]: Is that an errant asterisk after the file path? Should it be *_/var/lib/docker/_* ?

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: fixed! thanks!

**Comment by ahardin-rh** [comment]
Verified by Johnny Liu: https://bugzilla.redhat.com/show_bug.cgi?id=1296333

**Comment by ahardin-rh** [comment]
Also verified by Peter Larsen.

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -783,6 +789,9 @@ This will destroy any Docker containers or images currently on the host.
 # rm -rf /var/lib/docker/*
 # systemctl restart docker
 ----
++
+If there is any content in *_/var/lib/docker/_*, it has to be deleted. Files
```
Comment (adellape) [inline install_config/install/prerequisites.adoc]: Style thing:

s/has to/must/

File: install_config/install/prerequisites.adoc [hunk 3]
```diff
@@ -783,6 +789,9 @@ This will destroy any Docker containers or images currently on the host.
...
+will be present if Docker has been used prior to the install of OpenShift.
```
Comment (adellape) [inline install_config/install/prerequisites.adoc]: s/install/installation/

IBM Style Guide: use "install" as verb, "installation" for noun/adj.

**Comment by ahardin-rh** [comment]
**revhistory note:**
Updated to include guidance on how to check if Docker is running

**Pull request merged**